src/posts/helsinki-hashicorp-user-group-meetup-5.md

```diff
@@ -0,0 +1,38 @@
+---
+type: Event
+title: 'Helsinki HashiCorp User Group Meetup #5'
+subheading: Join us on 30 November at 17:00 for the fifth Helsinki HashiCorp User Group meetup!
+authors:
+- ckurowski
+tags:
+- HashiCorp
+date: 2022-28-10
+image: "/blogs/helsinki-hug-5/hug-blog-event-5.png"
+featured: false
+
+---
+
+**Helsinki HashiCorp User Group (HUG) is a quarterly meetup full of interesting talks and insights about all things HashiCorp, plus a chance to connect with fellow professionals and hobbyists alike.**
+
+**Time:** Wednesday, 30 November, 17:00-22.00 EET
+
+**Place:** UpCloud, Aleksanterinkatu 15 B, Helsinki
+
+[Register here](https://www.meetup.com/helsinki-hashicorp-user-group/events/288940115/)
+
+
+The next Helsinki HashiCorp User Group is hosted by [UpCloud](https://upcloud.com/) at their offices in downtown Helsinki. This time round there’ll be a packed agenda with four presentations and some very special guests. Join for an evening of talks, networking, and snacks! The talks will also be live streamed and recorded for those who prefer to join online.
```

Comment:
Amir's talk isn't really a presentation, but more of welcoming words & greetings.

Reply:
I can rephrase, so it's after the speakers - 'Plus some welcoming words from...'